plugins/tech-radar/src/components/RadarLegend/RadarLegend.tsx

```diff
@@ -32,11 +32,22 @@ export type Props = {
 };
 
 const useStyles = makeStyles<Theme>(theme => ({
+  foreignObject: {
```

Comment:
hm how about calling it `quadrantLegend` or something similar to make it more descriptive?